app/controllers/v3/service_route_bindings_controller.rb

```diff
@@ -85,6 +85,9 @@ def destroy
 
   def parameters
     route_binding_not_found! unless @route_binding && can_read_from_space?(@route_binding.route.space)
+    if @route_binding.create_initial? || @route_binding.create_failed?
```

Comment:
The check in the service credential bindings controller is `unless create_succeeded?` Thus I would prefer the same check here.